test/test_jit.py

```diff
@@ -1045,6 +1045,35 @@ def fn(*args):
                 self.assertFalse(fn.has_trace_for(*unk_config))
         self.assertEqual(fn.hits, 0)
 
+    def test_torch_sum(self):
```

Comment:
Is `torch.sum` supported added in this PR?

Reply:
Yes, torch.sum is supported in this PR.